src/hostapi/wmme/pa_win_wmme.c

```diff
@@ -298,45 +290,25 @@ static signed long GetStreamWriteAvailable( PaStream* stream );
 
 /* macros for setting last host error information */
 
-#ifdef UNICODE
-
-#define PA_MME_SET_LAST_WAVEIN_ERROR( mmresult ) \
-    {                                                                   \
-        wchar_t mmeErrorTextWide[ MAXERRORLENGTH ];                     \
-        char mmeErrorText[ MAXERRORLENGTH ];                            \
-        waveInGetErrorText( mmresult, mmeErrorTextWide, MAXERRORLENGTH );   \
-        WideCharToMultiByte( CP_ACP, WC_COMPOSITECHECK | WC_DEFAULTCHAR,\
-            mmeErrorTextWide, -1, mmeErrorText, MAXERRORLENGTH, NULL, NULL );  \
-        PaUtil_SetLastHostErrorInfo( paMME, mmresult, mmeErrorText );   \
+#define PA_MME_SET_LAST_WAVEIN_ERROR( mmresult )                              \
+    {                                                                         \
+        wchar_t mmeErrorTextWide[ MAXERRORLENGTH ];                           \
+        char mmeErrorText[ MAXERRORLENGTH ];                                  \
+        waveInGetErrorTextW( mmresult, mmeErrorTextWide, MAXERRORLENGTH );    \
+        WideCharToMultiByte( CP_UTF8, 0, mmeErrorTextWide, -1,                \
+            mmeErrorText, MAXERRORLENGTH, NULL, NULL );                       \
+        PaUtil_SetLastHostErrorInfo( paMME, mmresult, mmeErrorText );         \
     }
 
-#define PA_MME_SET_LAST_WAVEOUT_ERROR( mmresult ) \
-    {                                                                   \
-        wchar_t mmeErrorTextWide[ MAXERRORLENGTH ];                     \
-        char mmeErrorText[ MAXERRORLENGTH ];                            \
-        waveOutGetErrorText( mmresult, mmeErrorTextWide, MAXERRORLENGTH );  \
-        WideCharToMultiByte( CP_ACP, WC_COMPOSITECHECK | WC_DEFAULTCHAR,\
-            mmeErrorTextWide, -1, mmeErrorText, MAXERRORLENGTH, NULL, NULL );  \
-        PaUtil_SetLastHostErrorInfo( paMME, mmresult, mmeErrorText );   \
+#define PA_MME_SET_LAST_WAVEOUT_ERROR( mmresult )                             \
+    {                                                                         \
+        wchar_t mmeErrorTextWide[ MAXERRORLENGTH ];                           \
+        char mmeErrorText[ MAXERRORLENGTH ];                                  \
```

Comment:
As above, if an error is less than `MAXERRORLENGTH` wide characters long is it necessarily less than `MAXERRORLENGTH` bytes long?